tests/svg.js

```diff
@@ -0,0 +1,69 @@
+const fs = require('fs');
+const {DOMParser} = require('xmldom');
+const xpath = require('xpath');
+const core = require('@actions/core');
+let errors = false;
+
+// Function to test the SVG content against an XPath expression
+function test(svgContent, xpathExpression) {
+  try {
+    const doc = new DOMParser().parseFromString(svgContent, 'application/xml');
+    const nodes = xpath.select(xpathExpression, doc);
+    return nodes.length > 0;
+  } catch (err) {
+    core.error(`Failed to parse SVG content: ${err.message}`);
+    return false;
+  }
+}
+
+// Function to handle file checking
+async function main() {
+  const files = process.argv.slice(2);
+
+  await Promise.allSettled(files.map(async (file) => {
+    const error = (msg) => {
+      core.error(msg, {file});
+      errors = true;
+    };
+
+    const warn = (msg) => {
+      core.warning(msg, {file});
+    };
```

Comment:
Would it be better to have these functions outside the `Promise.allSettled` call?

Reply:
I have no idea what I'm doing @hkamran80 😂
Could you make the change? 😅 You're able to push to my repo directly

Reply:
I think the reason why I added them inside the function was so that I didn't have to provide the file as an argument.

Reply:
Ah. Makes sense.